prometheus.go

```diff
@@ -8,12 +8,28 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+const streamIDLabel = "streamID"
+
 var (
 	// mailboxCount tracks the current number of active mailboxes.
 	mailboxCount = prometheus.NewGauge(prometheus.GaugeOpts{
 		Namespace: "hashmail",
 		Name:      "mailbox_count",
 	})
+
+	// mailboxReadCount counts each time a mailbox pair is being used.
+	// A session consists of a bidirectional stream each using a mailbox
+	// with an ID that overlaps for the first 63 bytes and differ for the
+	// last byte. So in order to obtain accurate data about a specific
```

Comment:
micro nit: It's only the last "bit", not last byte.

Reply:
indeed! fixed 👍